files/cron.daily/chkrootkit

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+
+set -e
+
+CHKROOTKIT=/usr/sbin/chkrootkit
+CF=/etc/chkrootkit.conf
+LOG_DIR=/var/log/chkrootkit
+
+if [ ! -x $CHKROOTKIT ]; then
+  exit 0
+fi
+
+if [ -f $CF ]; then
+  . $CF
+fi
+
+TESTS=$(chkrootkit -l 2>&1 | cut -d ' ' -f 3-)
+
+# Remove disabled tests from list
+if [ -z $DISABLED_TESTS ]; then
```

Comment:
I'm not sure I understand the intention here.  The test `-z` returns true if the argument is the empty string, so this would only run the loop if there are no disabled tests, which does not seem to make sense.  Did you mean to use `-n` instead?

Moreover, this would result in a syntax error if there is more than one disabled test, so you need to add double quotes around the variable.